src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25409.cs

```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
```

Comment:
This specific test only have sense on iOS, right? Use compilation directives here.